src/python/WMCore/Services/PhEDEx/PhEDEx_DBS.py

```diff
@@ -0,0 +1,43 @@
+"""
+Module which depends on both PhEDEx and DBS.
+It provides function to create XML struction for injection.
+"""
+
+from __future__ import division
+
+from WMCore.Services.PhEDEx.XMLDrop import XMLInjectionSpec
+from WMCore.Services.DBS.DBSReader import DBSReader
+
+
+def makePhEDExDrop(dbsUrl, datasetPath, *blockNames):
+    """
+    _makePhEDExDrop_
+
+    Given a DBS Url, dataset name and list of blockNames,
+    generate an XML structure for injection
+
+    """
+    spec = XMLInjectionSpec(dbsUrl)
+
+
+    reader = DBSReader(dbsUrl, version = "DBS_2_0_9")
```

Comment:
The same as above change to DBS3Reader(dbsUrl), we should skip to passing DBS version so it will pick up lated one.